data/interfaces/default/graphs.html

```diff
@@ -258,9 +335,10 @@
             var m = dateValue.getMonth() + 1;
             var y = dateValue.getFullYear();
             var dateString = '' + y + '-' + (m<=9 ? '0' + m : m) + '-' + (d <= 9 ? '0' + d : d);
-
+            var u = '';
+            //var u = $.cookie("example") == 'tabs4') ? 'get_addedat_modal': 'history_table_modal'
             $.ajax({
-                "url": "history_table_modal",
+                "url": "get_addedat_modal",
```

Comment:
@Hellowlol possible a timezone issue between the server and client? If the graph is basing the data off of the JS `Date()` and the server is giving it based off of the server time/date then you'll have it not matching.

You're probably running the tests from the same server/timezone as your client.